app.rb

```diff
@@ -0,0 +1,97 @@
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def create_book(title, author)
+    puts
+    book = Book.new(title, author)
+    @books.push(book)
+    puts 'Book created successfully'
+    puts
+  end
+
+  def list_books
+    puts
+    if @books.empty?
+      puts 'No books added yet'
+    else
+      @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+    end
+    puts
+  end
+
+  def list_people
+    puts
+    if @people.empty?
+      puts 'No people added yet'
+    else
+      @people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
+    puts
+  end
+
+  def create_student(classroom, age, name, parent_permission)
+    student = Student.new(classroom, age, name: name, parent_permission: parent_permission)
+    puts
+    @people.push(student)
+    puts 'Person created successfully'
+    puts
+  end
+
+  def create_teacher(age, name, specialization)
+    teacher = Teacher.new(age, specialization, name: name)
+    @people.push(teacher)
+    puts
+    puts 'Person created successfully'
+    puts
```

Comment:
- I would advise you to fix this mistake and submit it again for re-review.

![Capture](https://user-images.githubusercontent.com/63972670/216728756-8acff525-1744-411b-9541-d80da7fd5ddc.JPG)